pkg/handlers/search_symbols.go

```diff
@@ -0,0 +1,68 @@
+package handlers
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/artmoskvin/hide/pkg/project"
+)
+
+const MinLimit = 1
+const MaxLimit = 100
+const DefaultLimit = 10
+
+type SearchSymbolsHandler struct {
+	pm project.Manager
+}
+
+func NewSearchSymbolsHandler(pm project.Manager) SearchSymbolsHandler {
```

Comment:
Let's add some options to the initialiser like `limit := DefaultLimit`, I prefer to have those upfront on initialisation also can make the service more configurable once we get to the config. [Functional Options Pattern](https://matheuspolitano.medium.com/unlocking-the-power-of-functional-options-pattern-in-go-087478f57be9) is nice, I typically use it but in this case struct in `SearchSymbolsHandler` will do.

Reply:
Done